app/models/assessment.rb

```diff
@@ -0,0 +1,11 @@
+class Assessment < ApplicationRecord
+  has_many :versions
+  belongs_to :organization
+  belongs_to :license
+
+  has_many :versions,
+            -> { order(created_at: :desc) },
+            inverse_of: :assessment
```

Comment:
Not that we have to have implement them right now, but can you go ahead and have a rough plan for how we will store variants of a dynamic/coded problem? (e.g. store the 1000 iterations of a randomzied problem).  Also need a plan for storing solutions both from the original poster and from others.  See `solution_key` later in a migration but don't know how that works.